kext/FileNVRAM/FileNVRAM.h

```diff
@@ -154,7 +154,10 @@ class FileNVRAM : public IODTNVRAM
     
     static IOReturn dispatchCommand( OSObject* owner, void* arg0, void* arg1, void* arg2, void* arg3 );
     
+    virtual IOReturn write_buffer(char* buffer, vfs_context_t ctx);
```

Comment:
mCtx is a member variable. the ctx parameter can be removed now that it's part of the class.